src/seedu/addressbook/ui/TextUi.java

```diff
@@ -156,13 +161,7 @@ private static String getIndexedListForViewing(List<String> listItems) {
         return formatted.toString();
     }
 
-    /**
-     * Formats a string as a viewable indexed list item.
-     *
-     * @param visibleIndex visible index for this listing
-     */
-    private static String getIndexedListItem(int visibleIndex, String listItem) {
-        return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
-    }
-
-}
+	private static String getIndexedListItem(int displayIndex, String listItem) {
+		return String.format(MESSAGE_INDEXED_LIST_ITEM, displayIndex, listItem);
+	}
+}
```

Comment:
on comments
Perhaps more formatting-related stuff (e.g. `DIVIDER`) can be moved to the `Formatter`? May be even things like this should be moved to the Formatter? Note that it may not always be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.